component/vshn_postgres.jsonnet

```diff
@@ -333,9 +333,7 @@ local composition =
     {
       apiVersion: 'appcat.vshn.io/v1',
       kind: 'XObjectBucket',
-      metadata: {
-        name: '',
-      },
+      metadata: {},
```

Comment:
Nit: from a crossplane perspective if `metadata.name` gets patched anyway you can omit the whole object.

No idea what would happen with the argo diff then.

Reply:
Hmm not sure about the argodiff. Currently it reports a `metadata: {}` so i took that. Both would probably work.

Reply:
Feel free to leave as is, it's just a nit :)